dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -1,8 +1,34 @@
+require 'json'
+require 'jwt'
 require "test_helper"
 
 class LtiV1ControllerTest < ActionDispatch::IntegrationTest
   setup_all do
     @integration = create :lti_integration
+    # create an arbitrary key for testing JWTs
+    @key = SecureRandom.alphanumeric 10
+  end
+
+  def create_jwt(payload)
+    JWT.encode(payload, @key)
+  end
+
+  def create_valid_jwt(aud_is_array)
+    target_link_uri = @integration.auth_redirect_url
```

Comment:
Wondering if using the `@integration.auth_redirect_url` here as a value might be confusing. It technically works and already exists, but functionally `target_link_uri` wouldn't ever be this value. Maybe we should consider using something like `CDO.studio_url('/', CDO.default_scheme)`? Or even just hardcode a string value?

Reply:
Yeah I just used it as an easy placeholder URL, I can replace it with the `studio_url` one.